test/Grpc.Core.Api.Tests/Grpc.Core.Api.Tests.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net45;netcoreapp3.1</TargetFrameworks>
```

Comment:
net45 -> net462

I think eventually these new projects should test with the same frameworks as existing test projects (e.g. + net5.0 and net6.0) but can leave it unchanged for now.